src/test/java/bio/terra/service/snapshot/SnapshotDaoTest.java

```diff
@@ -265,16 +267,10 @@ public void happyInOutTest() {
             .filter(t -> t.getName().equals("thetable"))
             .findFirst()
             .orElseThrow(AssertionError::new);
-    Table snapshotTable1 =
-        snapshot.getTables().stream()
-            .filter(t -> t.getName().equals("thetable"))
-            .findFirst()
-            .orElseThrow(AssertionError::new);
-    Table snapshotTable2 =
-        snapshot.getTables().stream()
-            .filter(t -> t.getName().equals("anothertable"))
-            .findFirst()
-            .orElseThrow(AssertionError::new);
+    SnapshotTable snapshotTable1 =
+        fromDb.getTableByName("thetable").orElseThrow(AssertionError::new);
```

Comment:
Why use a custom exception here? 
```suggestion
        fromDb.getTableByName("thetable").orElseThrow();
```